### PR TITLE
Readds Cat emotes

### PR DESCRIPTION
**Pull request by JamieD1**
Fuck off alexkar, you cant just close and revert a PR within 14 minutes then use the excuse of a exploit to do it.

Made the sounds quieter and added 10 second cooldown to meow.

Sounds are free to reuse.

🆑
tweak: Fixes Meow sound
/🆑

### CODE REVIEW

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -87,6 +89,24 @@
...
 
+/datum/emote/living/carbon/meow
+	key = "meow"
+	key_third_person = "meows"
+	message = "meows."
+	emote_type = EMOTE_AUDIBLE
+	cooldown = 10 SECONDS
+
+/datum/emote/living/carbon/meow/can_run_emote(mob/living/user, status_check = TRUE, intentional)
+	if(!ishuman(user))
```
Comment (adamsong) [inline code/modules/mob/living/carbon/human/emote.dm]: iscatperson already checks for human

**Review by adamsong — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -87,6 +89,24 @@
...
+		return
+	var/mob/living/carbon/human/H = user
+	if(iscatperson(H))
+		return TRUE
+	else
+		return FALSE
```
Comment (JamieD1) [inline code/modules/mob/living/carbon/human/emote.dm]:
```suggestion
		return iscatperson(user)
```

File: code/modules/mob/living/carbon/human/emote.dm [hunk 3]
```diff
@@ -87,6 +89,18 @@
...
 
+/datum/emote/living/carbon/meow
+	key = "meow"
+	key_third_person = "meows"
+	message = "meows."
+	emote_type = EMOTE_AUDIBLE
+	cooldown = 10 SECONDS
+
+/datum/emote/living/carbon/meow/can_run_emote(mob/living/user, status_check = TRUE, intentional)
+		return iscatperson(user)
```
Comment (JamieD1) [inline code/modules/mob/living/carbon/human/emote.dm]:
```suggestion
	return iscatperson(user)
```

**Review by SomeguyManperson — APPROVED** [review]
should work

**Review by adamsong — APPROVED** [review]
Code looks good

**Comment by JamieD1** [comment]
Waiting 24 HOURS as REQUIRED by all maintainers.

**Comment by alexkar598** [comment]
"Waiting 24 hours"

immediately testmerges it

Doesnt need testmerging

Basically a speed merge in disguise. Nice one Jamie

**Comment by adamsong** [comment]
This PR was open for more time before being test merged, than the other one which you full merged

**Comment by Marmio64** [comment]
even anime catgirls dont meow like a stray cat though

**Comment by Tipy1802** [comment]
Purrbation already makes a cat noise when triggered so cat sounds arent out of line

**Pull request merged**